test/lib/tasks/generateLibraryManifest.js

```diff
@@ -38,7 +37,7 @@ async function assertCreatedManifest(t, oExpectedManifest) {
 
 	await Promise.all(resources.map((resource) => workspace.write(resource)));
 
-	await tasks.generateLibraryManifest({
+	await generateLibraryManifest({
 		workspace: workspace,
```

Comment:
in other files touched by this change, you migrated to the short notation?

Reply:
Not sure what you refer to here. What's shorter than `await generateLibraryManifest({...})`?

Reply:
I'm missing the Gerrit highlighting :-)

I referred to 
```
  workspace: workspace,
  dependencies: dependencies,
```

Reply:
I see, done 🙂